packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -1923,6 +1929,64 @@ function updateContextConsumer(
   return workInProgress.child;
 }
 
+function updateEventComponent(current, workInProgress, renderExpirationTime) {
+  const nextProps = workInProgress.pendingProps;
+  let nextChildren = nextProps.children;
+
+  reconcileChildren(
+    current,
+    workInProgress,
+    nextChildren,
+    renderExpirationTime,
+  );
+  if (__DEV__) {
+    // We need to validate the event component does not have any direct children
+    // that are text host nodes (once resolved). So we use host context in the
+    // renderer and validate this inside the renderer. We only do this in DEV.
+    pushHostContext(workInProgress);
+  }
+  return workInProgress.child;
+}
+
+function updateEventTarget(current, workInProgress, renderExpirationTime) {
+  const nextProps = workInProgress.pendingProps;
+  let nextChildren = nextProps.children;
+
+  reconcileChildren(
+    current,
+    workInProgress,
+    nextChildren,
+    renderExpirationTime,
+  );
+  const parent = workInProgress.return;
+  invariant(
+    parent !== null && parent.tag === Event,
+    'Event target components must be direct children of event components',
+  );
+  const eventTargetType = nextProps.type;
+  // These warnings only occur in DEV to reduce overhead in production
+  if (__DEV__ && eventTargetType === REACT_EVENT_TARGET_TOUCH_HIT) {
```

Comment:
This can't happen at all right now since nothing uses the Symbol. Can we get a test of how you expect this to be used even if it's a light version of some internal implementation details of the plugin?